CONTRIBUTING.md

```diff
@@ -127,25 +129,47 @@ Excellent [PR guidelines](https://github.com/kubernetes/community/blob/master/co
 You will typically receive feedback from other developers. Before responding, take a moment to review the 
 [Code of Conduct](https://github.com/decred/dcrdata/blob/master/CODE_OF_CONDUCT.md). 
 
-Work through the comments and resolve any confusion with others and then implement the changes necessary.
+Work through the comments and resolve any confusion with others. Make whatever revisions are necessary.
 
 11. Resubmitting
 
-Similar to step #6 above, you will want to clean up your commits before creating the pull request. 
-As long as it hasn't been a very long time, you don't need to rebase to master. Instead, rebase 
-to the commit of the last pull request. To see a list of commits, you can use
+Commit your work as in step 5 above.
+
+###### a)
+
+Before resubmitting, clean up any little touchip commits you've made since the last time you pushed.
+If you've only made one commit since then, you can skip this step.
+For example, if you have made 3 commits since your last push, then run the following to put them together.
+
+```sh
+git rebase -i HEAD~3
+```
+
+The number after the tilda (~) is the number of commits that you want to combine, including the one you did at the beginning of this step.

```

Comment:
This would be a good place to note that if commits are being added after a github review, do not squash pre-review commits with post-review commits as having them separate allows github to show the reviewer relevant changes more easily.